examples/crm/src/deals/DealColumn.tsx

```diff
@@ -11,49 +11,72 @@ export const DealColumn = ({
 }: {
     stage: string;
     deals: Deal[];
-}) => (
-    <Box
-        sx={{
-            flex: 1,
-            paddingTop: '8px',
-            paddingBottom: '16px',
-            bgcolor: '#eaeaee',
-            '&:first-child': {
-                paddingLeft: '5px',
-                borderTopLeftRadius: 5,
-            },
-            '&:last-child': {
-                paddingRight: '5px',
-                borderTopRightRadius: 5,
-            },
-        }}
-    >
-        <Typography align="center" variant="subtitle1">
-            {/* @ts-ignore */}
-            {stageNames[stage]}
-        </Typography>
-        <Droppable droppableId={stage}>
-            {(droppableProvided, snapshot) => (
-                <Box
-                    ref={droppableProvided.innerRef}
-                    {...droppableProvided.droppableProps}
-                    className={snapshot.isDraggingOver ? ' isDraggingOver' : ''}
-                    sx={{
-                        display: 'flex',
-                        flexDirection: 'column',
-                        borderRadius: 1,
-                        padding: '5px',
-                        '&.isDraggingOver': {
-                            bgcolor: '#dadadf',
-                        },
-                    }}
+}) => {
+    const totalAmount = deals.reduce((sum, deal) => sum + deal.amount, 0);
+
+    return (
+        <Box
+            sx={{
+                flex: 1,
+                paddingTop: '8px',
+                paddingBottom: '16px',
+                bgcolor: '#eaeaee',
+                '&:first-child': {
+                    paddingLeft: '5px',
+                    borderTopLeftRadius: 5,
+                },
+                '&:last-child': {
+                    paddingRight: '5px',
+                    borderTopRightRadius: 5,
+                },
+            }}
+        >
+            <Stack alignItems="center">
+                <Typography variant="subtitle1">
+                    {/* @ts-ignore */}
+                    {stageNames[stage]}
+                </Typography>
+                <Typography
+                    variant="subtitle1"
+                    color="text.secondary"
+                    fontSize="small"
                 >
-                    {deals.map((deal, index) => (
-                        <DealCard key={deal.id} deal={deal} index={index} />
-                    ))}
-                    {droppableProvided.placeholder}
-                </Box>
-            )}
-        </Droppable>
-    </Box>
-);
+                    (
+                    {totalAmount.toLocaleString('en-US', {
```

Comment:
**non-blocking(nit-pick)**: Maybe this should be configurable in a config file or based on user's brower preference